src/spec/integration/inventories-sync.spec.coffee

```diff
@@ -103,6 +103,75 @@ describe 'Integration Inventories Sync', ->
     .catch (error) -> done(_.prettify(error))
   , 60000
 
+  it 'should add restockableInDays', (done) ->
+    ie =
+      sku: 'xyz1'
+      quantityOnStock: 10
+      version: 1
+    ieChanged =
+      sku: 'xyz1'
+      quantityOnStock: 10
+      restockableInDays: 30
+      version: 1
+    @client.inventoryEntries.create(ie)
+    .then (result) =>
+      expect(result.statusCode).toBe 201
```

Comment:
I think it would make sense to add an assertion here just to make sure `restockableInDays` is undefined.